Services/Tree/classes/class.ilMaterializedPathTree.php

```diff
@@ -226,7 +226,7 @@ public function insertNode($a_node_id, $a_parent_id, $a_pos)
 
 			$r = $ilDB->fetchObject($res);
 
-			if ($r->parent == NULL)
+			if ($r->parent === NULL)
```

Comment:
@smeyer-ilias could you please review these changes in the Tree-Classes? Thank you very much